srcpkgs/tlp/template

```diff
@@ -35,3 +35,21 @@ do_install() {
 
 	rm -rf ${DESTDIR}/etc/init.d
 }
+
+tlp-rdw_package() {
+	short_desc="Control the radio device wizard"
+	archs=noarch
+	depends="tlp NetworkManager"
+	pkg_install() {
+		make DESTDIR=${PKGDESTDIR} \
+		    TLP_SBIN=/usr/bin \
+		    TLP_ULIB=/usr/lib/udev \
+		    TLP_SHCPL=/usr/share/bash-completion/completions \
+		    TLP_ELOD=/usr/lib/elogind/system-sleep \
+		    TLP_WITH_SYSTEMD=0 \
+		    install-rdw
+
+		vmkdir usr/share/man/man8
```

Comment:
The convention is install everything in `do_install` step.

Then move sub-package's files in `pkg_install`.

It'll be easier for inspect.

Reply:
Yes this makes much more sense, I think I mis-understood how sub-packages are built and installed on using xbps-src. I hope the latest push is what you meant.